recipe/meta.yaml

```diff
@@ -154,15 +151,14 @@ outputs:
         - aiohttp >=3.7
         - aiohttp-cors
         - colorful
-        - gpustat >=1.0.0
+        - grpcio >=1.42.0
         - opencensus
         - prometheus_client >=0.7.1,<0.14.0
-        - pydantic <2.0
-        - jsonschema
+        - py-spy >=0.2.0
+        - pydantic !=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,<3
         - requests
         - smart_open
-        - py-spy >=0.2.0
-        - virtualenv >=20.0.24,<20.21.1
+        - virtualenv >=20.0.24,!=20.21.1
```

Comment:
It looks like a new package `memray` is required https://github.com/ray-project/ray/blob/ray-2.37.0/python/setup.py#L263C14-L263C20

Reply:
It was determined that `memray` is optional during refinement through an upstream code review.